third_party/silabs/SiWx917_sdk.gni

```diff
@@ -271,8 +279,12 @@ template("siwx917_sdk") {
 
     if (chip_enable_icd_server) {
       defines += [
-        "SL_CATALOG_POWER_MANAGER_PRESENT",
-        "SL_CATALOG_SLEEPTIMER_PRESENT",
+        "WIFI_DEBUG_ENABLED=1",
+        "SL_ICD_ENABLED=1",
+        "SL_ACTIVE_MODE_THRESHOLD=${sl_active_mode_threshold_ms}",
```

Comment:
did you add the associateed gn arguments?

Reply:
gn arguments are shared between the siwx917_sdk.gni and efr32_sdk.gni so it can be reused